tensorflow/core/distributed_runtime/rpc/grpc_worker_cache.cc

```diff
@@ -43,7 +43,7 @@ class GrpcWorkerCache : public WorkerCachePartial {
           void* tag;
           bool ok;
           while (completion_queue_.Next(&tag, &ok)) {
-            GrpcClientCQTag* callback_tag = static_cast<GrpcClientCQTag*>(tag);
+            auto callback_tag = static_cast<GrpcClientCQTag*>(tag);
```

Comment:
I have seen many examples for using `auto`, and here there is no need to write `GrpcClientCQTag*` with two times. So, I apply this change not only to use `auto` deduction, but to eliminate duplicate type declaration.